test/server/util-test.js

```diff
@@ -39,6 +41,29 @@ describe('Utils', () => {
         expect(escapeUserAgent('Chrome 67.0.3396 / Windows 8.1.0.0')).eql('Chrome_67.0.3396_Windows_8.1.0.0');
     });
 
+    it('Diff', () => {
+        expect(diff(null, null)).is.null;
```

Comment:
There are no server tests for checking argument  validation errors.

Reply:
I expect that result of some tests is null.
In this case using the throw/catch is just a way to stop function execution on the deep reccursion level.